lib/OrdinaryDiffEqDefault/Project.toml

```diff
@@ -16,6 +16,7 @@ OrdinaryDiffEqVerner = "79d7bb75-1356-48c1-b8c0-6832512096c2"
 PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
why is this needed as a dep?

Reply:
it could be test only, but since Rosenbrock depends on it explicitly, it already is a dep.